File: ThriftJavaDemo/ThriftDemoServer/src/main/java/com/qigang/thrift/impl/MyHelloService.java
package com.qigang.thrift.impl;

import com.qigang.thrift.HelloService;
import org.apache.thrift.TException;

public class MyHelloService implements HelloService.Iface {
    @Override
    public String HelloString(String para) throws TException {
        System.out.println("HelloString is called...");
        return para;
    }

    @Override
    public int HelloInt(int para) throws TException {
        System.out.println("HelloInt is called...");
        return para;
    }

    @Override
    public boolean HelloBoolean(boolean para) throws TException {
        System.out.println("HelloBoolean is called...");
        return para;
    }

    @Override
    public void HelloVoid() throws TException {
        System.out.println("HelloVoid is called...");
    }

    @Override
    public String HelloNull() throws TException {
        System.out.println("HelloNull is called...");
        return null;
    }
}

File: ThriftCSharpDemo/ThriftDemo.Server/MyHelloService.cs
using System;
using ThriftDemo.Interface;

namespace ThriftDemo.Server
{
    public class MyHelloService : HelloService.Iface
    {
        public bool HelloBoolean(bool para)
        {
            Console.WriteLine("客户端调用了HelloBoolean方法");

            return para;
        }

        public int HelloInt(int para)
        {
            Console.WriteLine("客户端调用了HelloInt方法");

            return para;
        }

        public string HelloNull()
        {
            Console.WriteLine("客户端调用了HelloNull方法");

            return null;
        }

        public string HelloString(string para)
        {
            Console.WriteLine("客户端调用了HelloString方法");

            return para;
        }

        public void HelloVoid()
        {
            Console.WriteLine("客户端调用了HelloVoid方法");

            Console.WriteLine("HelloWorld");
        }
    }
}
